File: Python/Users/GetUsers.py
import requests

boards_on_fire_api_url = "YOUR URL"  # example: https://my-company.boardsonfireapp.com/api/v5/
api_key = "YOUR API KEY"

headers = {
    "Content-Type": "application/json",
    "x-api-key": api_key
}

try:
    response = requests.get(f"{boards_on_fire_api_url}/users", headers=headers)
    response.raise_for_status()
    data = response.json()
    print("Users List:")
    print(data)
except requests.exceptions.RequestException as e:
    print(f"Request error: {e}")

File: .NET/Users/GetUsers.cs
using System.Net.Http.Headers;

var boardsOnFireApiUrl = "YOUR URL"; // example: https://my-company.boardsonfireapp.com/api/v5/
var apiKey = "YOUR API KEY";

using (var client = new HttpClient())
{
    client.BaseAddress = new Uri(boardsOnFireApiUrl);
    client.DefaultRequestHeaders.Accept.Clear();
    client.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
    client.DefaultRequestHeaders.Add("x-api-key", apiKey);

    try
    {
        HttpResponseMessage response = await client.GetAsync("users");
        response.EnsureSuccessStatusCode();

        var data = await response.Content.ReadAsStringAsync();
        Console.WriteLine("Users List:");
        Console.WriteLine(data);
    }
    catch (HttpRequestException e)
    {
        Console.WriteLine($"Request error: {e.Message}");
    }
}
